students/timm67/session08/circle.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+"""
+Tim Meese
+Au2018-Py210B
+Circle Lab
+"""
+
+from math import pi
+
+
+class Circle():
+    _radius = float(0.0)
+    _diameter = float(0.0)
+    _area = float(0.0)
+    _circumference = float(0.0)
+
+    def __init__(self, radius):
+        self._radius = float(radius)
+        self._diameter = float(2.0) * self._radius
+        self._area = pi * (self._radius * self._radius)
+        self._circumference = float(2.0) * pi * self._radius
+
+    @classmethod
+    def from_diameter(self, dia):
+        radius_in = dia // 2
```

Comment:
Minor: Hmm... Why integer division on `dia`?

Reply:
radius input is always an int and converted to float in constructor and subsequent calcs (just wanted to be consistent)